lib/rails/auth/x509/filter/pem.rb

```diff
@@ -5,7 +5,7 @@ module Filter
         # Extract OpenSSL::X509::Certificates from Privacy Enhanced Mail (PEM) certificates
         class Pem
           def call(pem)
-            OpenSSL::X509::Certificate.new(pem).freeze
+            OpenSSL::X509::Certificate.new(pem.delete("\t")).freeze
```

Comment:
Yeah. A pem with tabs produces a nested asn1 error. With the tabs removed it validates. I had been using a custom filter to do it, but wanted to start using the built-in one.